### ISSUE TITLE
SmoothLook only applied on the client-side

### ISSUE THREAD

**Issue by Bitlett** [cabaletta/baritone#4600]
## What do you need help with?
I was looking at the code of [LookBehavior](https://github.com/cabaletta/baritone/blob/1.19.4/src/main/java/baritone/behavior/LookBehavior.java) and happened to notice that SmoothLook is actually only applied on the client-side. This is the case because it is applied on the POST-end of the player update packet sending, rather than the PRE-end. I was just wondering whether this was intentional, as I personally can't see the point behind a feature like client-only SmoothLook when you can just enable FreeLook with Anti-Cheat Compatibility.

## Final checklist
- [x] I know how to properly use check boxes 
- [x] I have not used any OwO's or UwU's in this issue.

**Comment by ZacSharp** [comment]
Yes, this is intentional. The purpose of `smoothLook` is allowing you to watch Baritone without getting a headache, not actually altering behavior.
There is one case where it does influence actual rotations though: If you enable `smoothLook` and disable `freeLook`. Though it'd expect that to be unintentional.

**Comment by ZacSharp** [comment]
Oh, wait. Maybe I misunderstood you. Maybe you said that my second to last sentence is wrong.
Are you saying that even if Baritone itself uses the `smoothLook` rotation it nonetheless sends the target rotation to the server?

**Comment by Bitlett** [comment]
> Yes, this is intentional. The purpose of `smoothLook` is allowing you to watch Baritone without getting a headache, not actually altering behavior. There is one case where it does influence actual rotations though: If you enable `smoothLook` and disable `freeLook`. Though it'd expect that to be unintentional.

Got it. Thanks!
